robot/src/main/kotlin/org/team9432/robot/auto/autos/FourAllianceNote.kt

```diff
@@ -0,0 +1,34 @@
+package org.team9432.robot.auto.autos
+
+import edu.wpi.first.math.geometry.Rotation2d
+import org.team9432.lib.commandbased.commands.InstantCommand
+import org.team9432.lib.commandbased.commands.ParallelCommand
+import org.team9432.lib.commandbased.commands.SequentialCommand
+import org.team9432.robot.RobotState
+import org.team9432.robot.auto.AllianceNote
+import org.team9432.robot.auto.AutoConstants
+import org.team9432.robot.auto.commands.*
+import org.team9432.robot.auto.subsections.AlignToIntakeNote
+import org.team9432.robot.auto.subsections.IntakeNote
+import org.team9432.robot.commands.drivetrain.DriveToPosition
+
+fun FourAllianceNote() = SequentialCommand(
+    InitAuto(Rotation2d.fromDegrees(180.0)),
```

Comment:
Will do some more digging. But we seem to have a good number of places that we are performing the 180º rotation... Maybe I'm going crazy but I think cleaning this up will be helpful for code maintainability.